### PR TITLE
Fix untranslatable parts of the UI

### PR DESCRIPTION
**Pull request by ivanbratovic**
# Description

There are many untranslated parts of text (in English) appearing in the UI: buttons, placeholder texts, etc. (see attached screenshots). This PR should address **currently** untranslatable parts of the UI and make them translatable.

## Type of change

- User Interface

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and test it
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

### Examples of some untranslatable parts of the UI

Examples in Croatian so you can clearly see the English text. The Croatian language is up-to-date, so the parts of the UI in English are not translatable.

#### Shrink Database button
Shrink database

#### "Default" badge
Default

#### Placeholders and title in HTTP Options
Placeholders

### CODE REVIEW

**Comment by ivanbratovic** [comment]
I believe the changes are ready for review. Some untranslatable text was left in `server.js`, but I haven't found a way to use the i18n's `$t` function within it.

**Review by Saibamen — APPROVED** [review]

**Comment by Saibamen** [comment]
> I believe the changes are ready for review. Some untranslatable text was left in `server.js`, but I haven't found a way to use the i18n's `$t` function within it.

If you talking about messages for frontend, we need to rewrite it to contains keys like `SOME_ACTION_MESSAGE` and use it to add translations in frontend

File: src/pages/EditMonitor.vue [hunk 1]
```diff
@@ -352,20 +352,6 @@ export default {
...
-}`;
-        },
-
-        headersPlaceholder() {
-            return `Example:
-{
-    "HeaderName": "HeaderValue"
-}`;
-        }
-
```
Comment (louislam) [inline src/pages/EditMonitor.vue:368]: I prefer not to translate the json. Just need to translate "Example:"

Comment (ivanbratovic) [reply src/pages/EditMonitor.vue]: I reverted these changes, used the existing translation for "Example:" and kept the JSON placeholders as they were in `EditMonitor.vue`.

**Pull request merged**